commands/build.go

```diff
@@ -502,6 +743,9 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 
 	if isExperimental() {
 		flags.StringVar(&options.invoke, "invoke", "", "Invoke a command after the build [experimental]")
+		flags.StringVar(&options.root, "root", "", "Specify buildx server root [experimental]")
+		flags.BoolVar(&options.debug, "debug", false, "Print debug logs [experimental]")
+		flags.BoolVar(&options.detach, "detach", runtime.GOOS == "linux", "Detach buildx server (supported only on linux) [experimental]")
```

Comment:
What is the linux specific here? I assume macos shouldn't have much issues?

@crazy-max Maybe you can check windows viability?

Reply:
The location of buildx root dir and the logic to detach server  ([commands/launch_linux.go](https://github.com/docker/buildx/pull/1296/files#diff-32862d0205291fed261cec68090fd736f0990bbceb50f81fe8f626e58ec5f21c)) currently assumes linux.